imap_processing/codice/L0/p_cod_aut_test.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version='1.0' encoding='utf-8'?>
+<xtce:SpaceSystem xmlns:xtce="http://www.omg.org/space/xtce" nsmap="{'xtce': 'http://www.omg.org/space'}" name="P_COD_AUT">
```

Comment:
When we met yesterday, you were showing me how you had to manually go into this file to add the `xtce` part to the end of the URL.  If you want to implement a solution in your `make_xtce.py` module to do this for you, you could use something like this:

```python
with open('L0/p_cod_aut_test.xml', 'r') as file:
    contents = file.read()

modified_content = contents.replace('xmlns:xtce="http://www.omg.org/space/"', 'xmlns:xtce="http://www.omg.org/space/xtce"')

with open('L0/p_cod_aut_test.xml', 'w') as file:
    contents = file.write()
```

Reply:
I tried working this in, but I could not get it to work.